inc/template-tags.php

```diff
@@ -158,47 +158,36 @@ function primer_paging_nav() {
 /**
  * Display navigation to next/previous post when applicable.
  *
- * @global WP_Post $post
+ * @uses the_post_navigation
+ * @uses primer_post_nav_text
+ *
  * @since  1.0.0
  */
 function primer_post_nav() {
 
-	global $post;
-
-	$previous = is_attachment() ? get_post( $post->post_parent ) : get_adjacent_post( false, '', true );
-	$next     = get_adjacent_post( false, '', false );
-
-	if ( ! $next && ! $previous ) {
-
-		return;
-
-	}
-
-	?>
-	<nav class="navigation post-navigation">
-
-		<h1 class="screen-reader-text"><?php esc_html_e( 'Post navigation', 'primer' ); ?></h1>
-
-		<div class="nav-links">
-
-		<?php if ( is_rtl() ) : ?>
-
-			<div class="nav-next"><?php next_post_link( '%link &larr;' ); ?></div>
-
-			<div class="nav-previous"><?php previous_post_link( '&rarr; %link' ); ?></div>
-
-		<?php else : ?>
-
-			<div class="nav-previous"><?php previous_post_link( '&larr; %link' ); ?></div>
-
-			<div class="nav-next"><?php next_post_link( '%link &rarr;' ); ?></div>
-
-		<?php endif; ?>
+	/**
+	 * Filter the next post navigation label
+	 *
+	 * @since NEXT
+	 *
+	 * @var string
+	 */
+	$next_label = (string) apply_filters( 'primer_post_nav_label_next', '%title &rarr;' );
 
-		</div><!-- .nav-links -->
+	/**
+	 * Filter the prev post navigation label
+	 *
+	 * @since NEXT
+	 *
+	 * @var string
+	 */
+	$prev_label = (string) apply_filters( 'primer_post_nav_label_prev', '&larr; %title' );
 
-	</nav><!-- .navigation -->
-	<?php
+	the_post_navigation( array(
```

Comment:
@EvanHerman Upon digging further, it turns out we aren't supposed to pass this markup through the args anyway because it's handled by `get_the_post_navigation()` in core.